governance/votes/Executive vote - May 28, 2021.md

```diff
@@ -36,6 +39,22 @@ If this executive proposal does not pass within 30 days, then it will expire and
 
 ## Proposal Details
 
+### Increase PSM-USDC-A Debt Ceiling
+
+As per this urgent [signal thread](https://forum.makerdao.com/t/urgent-adjustments-to-tip-psm-usdc-a-dc/8274), the mandated actors have decided to propose increases to the PSM-USDC-A Debt Ceiling to help mitigate the possibility of DAI deviating from the dollar peg in the event of further volatility in the market. The PSM-USDC-A Debt Ceiling will increase from 2 billion DAI to 2 billion DAI if this executive proposal passes.
+
+The reasoning for this urgent inclusion can be found [here](https://forum.makerdao.com/t/urgent-adjustments-to-tip-psm-usdc-a-dc/8274/11).
+
+### Increase Flat Kick Incentive (`tip`) for All Vault Types
+
+As per this urgent [signal thread](https://forum.makerdao.com/t/urgent-adjustments-to-tip-psm-usdc-a-dc/8274), the mandated actors have decided to propose setting the [Flat Kick Incentive (`tip`)](https://community-development.makerdao.com/en/learn/governance/param-flat-kick-incentive) for all vaults to 300 DAI to ensure liquidations run smoothly for smaller vaults in the event of further volatility in the market. The [Flat Kick Incentive](https://community-development.makerdao.com/en/learn/governance/param-flat-kick-incentive) will increase from 0 DAI to 300 DAI on all vault types with liquidations enabled if this executive proposal passes.
```

Comment:
Seems a little wordy? Maybe:

As per this urgent [signal thread](https://forum.makerdao.com/t/urgent-adjustments-to-tip-psm-usdc-a-dc/8274), the mandated actors have decided to propose changing the [Flat Kick Incentive (`tip`)](https://community-development.makerdao.com/en/learn/governance/param-flat-kick-incentive) to ensure liquidations run smoothly for smaller vaults in the event of further volatility in the market. The [Flat Kick Incentive](https://community-development.makerdao.com/en/learn/governance/param-flat-kick-incentive) will increase from 0 DAI to 300 DAI on all vault types with liquidations enabled if this executive proposal passes.